lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -373,7 +373,7 @@ dictionary LnUrlPayRequest {
 
 dictionary PrepareSendRequest {
     string destination;
-    u64? amount_sat = null;
+    PayOnchainAmount? amount = null;
```

Comment:
The name is a bit confusing because we use onchain for bitcoin.
do you think it makes sense to use amount_sat as before and add the drain option default to false?

Reply:
I feel like the drain option should be an explicit opt-in and would be more consistent with the drain to Bitcoin. I would rather rename the PayOnchainAmount to something like DrainableAmount. Happy to revert to use the amount_sat if you prefer.

Reply:
Perhaps rename to SendPaymentAmount or PayAmount ?